File: petcare/lib/model-petcare/order-detail.dart
class OrderDetail {
  int? orderDetailId;
  int? orderId;
  int? productId;
  int? serviceId;
  int? quantity;
  double? price;

  OrderDetail(
      {this.orderDetailId,
      this.orderId,
      this.productId,
      this.serviceId,
      this.quantity,
      this.price});

  OrderDetail.fromJson(Map<String, dynamic> json) {
    orderDetailId = json['orderDetailId'];
    orderId = json['orderId'];
    productId = json['productId'];
    serviceId = json['serviceId'];
    quantity = json['quantity'];
    price = json['price'];
  }

  Map<String, dynamic> toJson() {
    final Map<String, dynamic> data = new Map<String, dynamic>();
    data['orderDetailId'] = this.orderDetailId;
    data['orderId'] = this.orderId;
    data['productId'] = this.productId;
    data['serviceId'] = this.serviceId;
    data['quantity'] = this.quantity;
    data['price'] = this.price;
    return data;
  }
}

File: pets4life-api/BusinessObject/Models/OrderDetail.cs
using System;
using System.Collections.Generic;
using System.Text.Json.Serialization;

namespace BusinessObject.Models
{
    public partial class OrderDetail
    {
        public int OrderDetailId { get; set; }
        public int? OrderId { get; set; }
        public int? ProductId { get; set; }
        public int? ServiceId { get; set; }
        public int? Quantity { get; set; }
        public decimal? Price { get; set; }

       [JsonIgnore]  public virtual Order? Order { get; set; }
        [JsonIgnore] public virtual Product? Product { get; set; }
        [JsonIgnore] public virtual Service? Service { get; set; }
    }
}
